server/controllers/calls.js

```diff
@@ -107,6 +108,33 @@ function afterPutCallAttempt(res, outcome, contact_id, attempt_num, campaign_id)
   }
   return res.status(200).json({ message: 'call log successfully updated' });
 }
+
+function handleHangUpFlow(res, user_id, call_id, newStatus) {
+  return usersService.getUserById({ id: user_id })
+  .then((userObj) => {
+    const { call_sid: userCallSid } = userObj.attributes;
+    console.log(userObj.attributes);
```

Comment:
Can we remove these console logs if we're done debugging?

Reply:
Yeah, sorry